esmvalcore/cmor/_fixes/native6/era5_land.py

```diff
@@ -0,0 +1,127 @@
+"""Fixes for ERA5-Land."""
+import datetime
+import logging
+
+import iris
+import numpy as np
+
+from esmvalcore.cmor._fixes.native6.era5 import (get_frequency,
+                                                 fix_hourly_time_coordinate,
+                                                 fix_accumulated_units,
+                                                 multiply_with_density,
+                                                 remove_time_coordinate,
+                                                 divide_by_gravity)
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+logger = logging.getLogger(__name__)
+
+
+class Orog(Fix):
+    """Fixes for orography."""
+
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        fixed_cubes = []
+        for cube in cubes:
+            cube = remove_time_coordinate(cube)
+            divide_by_gravity(cube)
+            fixed_cubes.append(cube)
+        return iris.cube.CubeList(fixed_cubes)
+
+
+class Pr(Fix):
```

Comment:
I'm a bit conflicted about code duplication. If the fixes are exactly the same, we might as well import everything from `esmvalcore.cmor._fixes.native6.era5`, no? What kind of differences do we expect?

Reply:
I was not sure if it is possible to import classes, therefore I left them in. But they could be imported if possible. The main difference we expect is, that there are some variables only in ERA5 and others only in ERA5-Land.

Reply:
I tested it now, it is possible and shortens the code to nearly nothing (one just has to use them, so I made a logger.info as kind of minimum use now).
Like this it is also easy to import more fixes from ERA5 as soon as somebody has time to test if they work for ERA5-Land as well.